src/components/Timer/getTime.ts

```diff
@@ -42,6 +45,9 @@ const getTimes = ({ path, challenge }: Props): number | null => {
   if (startLuxonTime === null || endLuxonTime === null) {
     return null
   }
-  return endLuxonTime.toMillis() - startLuxonTime.toMillis()
+  let luxonMilliseconds = endLuxonTime.toMillis() - startLuxonTime.toMillis()
+  return Duration.fromObject({ milliseconds: luxonMilliseconds }).toFormat(
+    'hh:mm:ss',
```

Comment:
Question: is there any reason why we aren't using a secondary function to convert milliseconds to strings? I think it would be better if this function returned a number of milliseconds, after Thomas was explaining it. Returning a string could work, but It'd be great to hear why that type is a better option.

Reply:
Unresolving this because it hasn't seemed to have changed yet, if you already have gotten some changes locally you can upload them and I'll un-mark this comment!